hs_core/models.py

```diff
@@ -871,7 +871,7 @@ class Date(AbstractMetaDataElement):
         ('available', 'Available')
     )
     HS_DATE_TYPE_CHOICES = (
-        ('reviewStarted', 'Review Started'),
+        ('review_started', 'Review Started'),
```

Comment:
Is this going to break tests? It seems there are test xml data files that use "reviewStarted"